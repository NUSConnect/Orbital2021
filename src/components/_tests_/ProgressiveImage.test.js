import * as React from 'react'
import ProgressiveImage from '../ProgressiveImage'
import renderer from 'react-test-renderer'

jest.useFakeTimers()

it('renders correctly', () => {
  const tree = renderer.create(<ProgressiveImage />)
  expect(tree).toMatchSnapshot()
})
