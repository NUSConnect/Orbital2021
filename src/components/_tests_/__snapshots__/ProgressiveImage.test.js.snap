// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#e1e4e8",
    }
  }
>
  <Image
    blurRadius={1}
    onLoad={[Function]}
    style={
      Object {
        "opacity": 0,
      }
    }
  />
  <Image
    onLoad={[Function]}
    style={
      Object {
        "bottom": 0,
        "left": 0,
        "opacity": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
      }
    }
  />
</View>
`;
